README.md

```diff
@@ -49,3 +49,21 @@ We encourage the use of any libraries for everything except the search functiona
 * Remember to make meaningful commits as you work
 * Somehow share your repository with us
 * __Important:__ If there are credentials required (.env or master.key file), please email these to us directly or we can’t review your project
+* After you fork the repo, you can run the following commands in your terminal:
+        
+        bin/setup
+        bin/rspec
+        
+* Feel free to add to add/modify existing specs. Just make sure the original request specs are still present.
```

Comment:
Should we add a note in the instructions that the routes and params should be named as per the existing request specs? I've seen a ton of variation in naming so I wouldn't want someone to start on a solution without realizing these have already been set.

Reply:
Agreed I think listing the routes is a small nudge to get them moving in the right direction and will probably make our lives easier!